frontend/pages/software/components/EmptySoftware.tsx

```diff
@@ -0,0 +1,68 @@
+// This component is used on ManageSoftwarePage.tsx and DashboardPage.tsx > Software.tsx card
```

Comment:
I don't see this in ManageSoftwarePage.tsx – are both this and EmptySoftwareTable.tsx used but in different places?

Reply:
This shows if your software is empty, if you had hosts and software, it might take up to 1 hour to respond as our dev environment checks for software hourly